drivers/ethernet/eth_stm32_hal.c

```diff
@@ -209,9 +209,10 @@ static int eth_tx(struct device *dev, struct net_pkt *pkt)
 	}
 
 #if defined(CONFIG_SOC_SERIES_STM32H7X)
-	ETH_BufferTypeDef tx_buffer_def[ETH_TXBUFNB];
+	/* Only one tx_buffer is sufficient to pass only 1 dma_buffer */
```

Comment:
I would put this comment above the line with `#define ETH_TXBUF_DEF_NB	1U`.

Reply:
You're right that has more sense. Will do the change asap

Reply:
Done !